### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by cerebrl**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,7 +60,6 @@
     "autoprefixer": "^10.2.6",
     "bootstrap": "^5.0.1",
     "copyfiles": "^2.1.1",
-    "cpy-cli": "^3.1.1",
```
Comment (cerebrl) [inline package.json:63]: Removed due to security alert.

File: package.json [hunk 2]
```diff
@@ -71,7 +70,7 @@
     "jest-junit": "^12.0.0",
     "jest-puppeteer": "^4.4.0",
     "miragejs": "^0.1.32",
-    "node-sass": "^4.14.1",
+    "node-sass": "^6.0.1",
```
Comment (cerebrl) [inline package.json:73]: Upgraded due to security alert.

File: webpack.config.scss.js [hunk 3]
```diff
@@ -9,8 +9,8 @@ module.exports = (env) => {
       apply: (compiler) => {
         compiler.hooks.afterEmit.tap('AfterEmitPlugin', (compilation) => {
           const cmds = [
-            'cpy ./bundles/fr-ui.css ./samples/_static/css',
-            'cpy ./bundles/fr-ui.css ./tests/e2e/app/_static/css',
+            'copyfiles -u 1 "./bundles/fr-ui.css" ./samples/_static/css/',
+            'copyfiles -u 1 "./bundles/fr-ui.css" ./tests/e2e/app/_static/css',
```
Comment (cerebrl) [inline webpack.config.scss.js:13]: Switched due to security alert.

**Review by spetrov — APPROVED** [review]
Looks good! 👍🏻

**Pull request merged**